tests/unit_tests/test_wave_06.py

```diff
@@ -76,16 +75,27 @@ def test_swap_best_by_category():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
+    # raise Exception("Complete this test according to comments below.")
     # *********************************************************************
     # ****** Complete Assert Portion of this test **********
     # *********************************************************************
     # Assertions should check:
     # - That the results is truthy
     # - That tai and jesse's inventories are the correct length
     # - That all the correct items are in tai and jesse's inventories, including the items which were swapped from one vendor to the other
+    assert result == True
+    # assert tai.get_best_by_category.category == "Clothing"
+    # assert tai.get_best_by_category.condition == pytest.approx(4.0)
+    # assert jesse.get_best_by_category.category == "Decor"
+    # assert jesse.get_best_by_category.condition == pytest.approx(4.0)
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
+    assert item_f in tai.inventory
+    assert item_c not in tai.inventory
+    assert item_f not in jesse.inventory
+    assert item_c in jesse.inventory
```

Comment:
Great start! It would also be a good idea to make sure the other items in Tai and Jesse's inventory are what we expect them to be.